src/__tests__/renderer.test.tsx

```diff
@@ -68,7 +68,11 @@ describe('slack jsx', () => {
 
   it('component with array of nested component children', () => {
     const message = (
-      <Message token="test_token" channel="test_channel">
+      <Message
+        token="test_token"
+        channel="test_channel"
+        altText={<AltText>this is so got</AltText>}
```

Comment:
so `got`?

Reply:
exactly why i shouldn't be coding this late. but fixed and updated tests accordingly